base_location_nuts_no_update/README.rst

```diff
@@ -0,0 +1,73 @@
+======================
```

Comment:
Can you use the seperated readme to generate the README, please?

Reply:
Hi @tarteo, thanks for your review. 
Sorry but I don't get what you are suggesting here, I already have generated this `README.rst` using [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)

Reply:
Oh yes, I see :stuck_out_tongue:. Oops